better-font-awesome.php

```diff
@@ -513,18 +543,67 @@ public function text_callback( $args ) {
      * @return  string  Usage text.
      */
     public function get_usage_text() {
```

Comment:
or maybe even you could add something like this? Even cooler - you could pass args? 
````php
function get_template( $template, $args ) {
    $path = __DIR__ . '/templates/usage-text.php'; 
    if ( file_exists( $path ) ) {
        ob_start();
        include $path;
        return ob_get_clean();
    }
}
````